catalog/src/main/kotlin/com/adevinta/spark/catalog/CatalogApp.kt

```diff
@@ -207,8 +208,17 @@ internal fun ComponentActivity.CatalogApp(
             ) {
                 val coroutineScope = rememberCoroutineScope()
                 val homeScreenValues = CatalogHomeScreen.entries
+                val initialPage = LocalActivity.current?.intent?.data?.let {
+                    when {
+                        it.pathSegments.contains("examples") -> CatalogHomeScreen.Examples.ordinal
+                        it.pathSegments.contains("configurator") -> CatalogHomeScreen.Configurator.ordinal
+                        it.pathSegments.contains("icons") -> CatalogHomeScreen.Icons.ordinal
```

Comment:
nit:
```suggestion
                        "examples" in it.pathSegments -> CatalogHomeScreen.Examples.ordinal
                        "configurator" in it.pathSegments -> CatalogHomeScreen.Configurator.ordinal
                        "icons" in it.pathSegments -> CatalogHomeScreen.Icons.ordinal
```

Reply:
Done in 7b20af4f02332c94d5dab6704f845e144ae35eb8